examples/exotica_examples/tests/collision_scene_distances.py

```diff
@@ -66,16 +67,17 @@ def testSphereVsSpherePenetrating(collisionScene):
 
     p = scene.getCollisionDistance("A", "B")
     debugPublish(p, scene)
+    print(p)
```

Comment:
Is this required?
Also in subsequent pints?

Reply:
Yes. I left it in the ones that frequently break to be able to spot more easily.